starter-code/scripts/models/repo.js

```diff
@@ -4,11 +4,22 @@
   var reposObj = {};
 
   reposObj.allRepos = [];
-// TODO: create a githubToken.js file that we can use to generate our headers
+// TODO: DONE: create a githubToken.js file that we can use to generate our headers
          // properly.
   reposObj.requestRepos = function(callback) {
-    /* TODO: How would you like to fetch your repos? Someone say AJAX?!
+    /* TODO: DONE: How would you like to fetch your repos? Someone say AJAX?!
       Do not forget to call the callback! */
+    $.ajax({
+      url: 'https://api.github.com/user/repos?type=owner',
+      method: 'GET',
+      headers: {
+        'Authorization':'token ' + githubToken},
+      success: function(data) {
+        // console.log(data);
+        reposObj.allRepos = data;
+        callback();
```

Comment:
Great work constructing your AJAX call with the appropriate URL, method, and headers!